lhotse/cut/base.py

```diff
@@ -394,6 +394,7 @@ def trim_to_supervisions(
         keep_overlapping: bool = True,
         min_duration: Optional[Seconds] = None,
         context_direction: Literal["center", "left", "right", "random"] = "center",
+        ignore_channel: bool = False,
```

Comment:
What do you think about `keep_all_channels: bool = False`? `ignore_channel` is a little vague for me, but I'm not sure.